ctail

```diff
@@ -16,28 +16,39 @@ if ! TAIL=$(which tail); then
     echo -e "Cannot find tail executable.\n"
     exit 1
 fi
+
 if ! AWK=$(which awk); then
     echo -e "Cannot find awk executable.\n"
     exit 1
 fi
 
+awk_opts=()
+# detect if awk is mawk (which is common on debian based distros)
+case "$(basename "$(readlink -f "$AWK")")" in
```

Comment:
On macOS `readlink` is actually another way to invoke `stat` and it's missing the `-f` option.

Maybe we should rely on https://formulae.brew.sh/formula/coreutils and then check on which OS the script is running.

Reply:
@raducotescu Thanks ... good catch!

Reply:
@raducotescu I do not have access to macOS at the moment.
Would you be able to test it for me please.

Reply:
The latest commit looks good. I'll squash and merge your changes. Thanks!